install-dev/data/xml/hook.xml

```diff
@@ -886,41 +886,6 @@
       <title>Perform actions before controller initialization</title>
       <description>This hook is launched before the initialization of all controllers</description>
     </hook>
-    <hook id="actionAdminLoginControllerBefore">
```

Comment:
@jolelievre do you plan to reintroduce those hooks? one way or another, I'm pretty sure they've been useful for some 2FA modules

Reply:
@jolelievre unless there are certain events triggered on the SF side of things that could be a replacement

Reply:
No these hooks were triggered by the legacy controller, the legacy controller is no more and it wouldn't make sense to maintain them because we can't dispatch the same parameters (which as the legacy controller)

Usually we actually never maintain/remove the hooks removed along with legacy pages, but this time I thought it as worth the extra effort This way I also noted which hooks were actually removed and I'm adding them in the dev doc

My plan is to explain which hook were removed, which ones were maintained (two actually), and what alternative hooks can be used no even if they are different, I'll ping you on the PR for the doc

If we realize some hooks are missing we can also think of what new hooks could/should be introduced

Reply:
There is one hook that was kept strictly for legacy retro compatibility here:
https://github.com/PrestaShop/PrestaShop/blob/29bf2477c2c5fac8b2d51f3dc05d80e46644f88e/src/PrestaShopBundle/Twig/Component/Login/LoginHeadTag.php#L38-L43

It does provide a legacy controller, or actually a "fake" legacy controller we implemented as a bridge to still allow some legacy modifications like `$legacyController->addJs`

Maintaining this hook to allow adding assets was relevant, but all the other hooks would be overkill IMHO It gives the wrong impression that nothing changed and it forces us to maintain retro compatibility in too many places